src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 Microsoft.CodeAnalysis.CSharp.LanguageVersion.CSharp7_2 = 702 -> Microsoft.CodeAnalysis.CSharp.LanguageVersion
+Microsoft.CodeAnalysis.CSharp.IConversionExpressionExtensions
+static Microsoft.CodeAnalysis.CSharp.IConversionExpressionExtensions.GetCSharpConversion(this Microsoft.CodeAnalysis.Semantics.IConversionExpression conversionExpression) -> Microsoft.CodeAnalysis.CSharp.Conversion
```

Comment:
I think the name would suffice as GetConversion.  We don't have ClassifyCSharpConversion and ClassifyVisualBasicConversion.  We just ahve the extension ClassifyConversion at the particular C# and VB layers.